crates/vm/levm/src/opcode_handlers/system.rs

```diff
@@ -157,12 +158,13 @@ impl VM {
     }
 
     // DELEGATECALL operation
+    // TODO: Add gas consumption
```

Comment:
Open an issue for this and link it.

Reply:
The same goes for each todo below.

Reply:
Done in [1ff2d77](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1029/commits/1ff2d77272f33be8879304c3e849b0b39c95df9d).